api/src/services/eml-service.ts

```diff
@@ -653,6 +626,28 @@ export class EmlService extends DBService {
     return additionalMetadata;
   }
 
+  async _buildPartnershipMetadata(projectData: IGetProject): Promise<any> {
+    const stakeholders = projectData.partnerships.stakeholder_partnerships;
+    const codes = await this.codes();
+    const indigenousPartnerships = projectData.partnerships.indigenous_partnerships;
+    const firstNationsNames = codes.first_nations
+      .filter((code) => indigenousPartnerships.includes(code.id))
+      .map((code) => code.name);
+
+    const sortedPartnerships = _.sortBy([...firstNationsNames, ...stakeholders]);
```

Comment:
Nit: Could we not just use `Array.sort`? If not, could we import `sortBy` from Lodash instead of importing the entire object?

Reply:
it was originally but the default `.sort()` call is considered a bug by sonar cloud. Sonar cloud requires adding a custom sort algorithm because the default behavior is to sort alphabetically (which is what I wanted). I figured using lodash here was a cleaner solution instead of just replicating the default sort pattern for the sake of sonar cloud.